test/unit/api/python/test_solver.py

```diff
@@ -0,0 +1,36 @@
+#####################
+## test_term.py
+## Top contributors (to current version):
+##   Yoni Zohar
+## This file is part of the CVC4 project.
+## Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+## in the top-level source directory and their institutional affiliations.
+## All rights reserved.  See the file COPYING in the top-level source
+## directory for licensing information.
+##
+import pytest
+import pycvc4
+import sys
+
+from pycvc4 import kinds
+
+class TestSolver:
+    def setup_class(self):
+        self.solver = pycvc4.Solver()
+
+    def test_recoverableException(self):
```

Comment:
Personal preference here, but I dislike the mix of underscore and camel case. It would also be more pythonic to just use underscores.